packages/client-deva/package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "@ai16z/client-deva",
+    "version": "0.1.4-alpha.3",
+    "main": "dist/index.js",
+    "type": "module",
+    "types": "dist/index.d.ts",
+    "dependencies": {
+        "@ai16z/eliza": "workspace:*",
+        "agent-twitter-client": "0.0.16",
```

Comment:
don't think this is needed

Reply:
Thank you for pointing this out, I removed the twitter client

Reply:
Can you please tell me what are the next steps if you know 
Do we create docs for this part, or should we just wait for an approval?